packages/@aws-cdk/aws-pipes-targets-alpha/lib/event-bridge.ts

```diff
@@ -0,0 +1,130 @@
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-detailtype
+   * @default - none
+   */
+  readonly detailType?: string;
+
+  /**
+   * The URL subdomain of the endpoint.
+   *
+   * @example
+   * // if the URL for the endpoint is https://abcde.veo.endpoints.event.amazonaws.com
+   * 'abcde.veo'
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-endpointid
+   * @default - none
+   */
+  readonly endpointId?: string;
+
+  /**
+   * AWS resources, identified by Amazon Resource Name (ARN), which the event primarily concerns.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-resources
+   * @default - none
+   */
+  readonly resources?: string[];
+
+  /**
+   * The source of the event.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-source
+   * @default - none
+   */
+  readonly source?: string;
+
+  /**
+   * The time stamp of the event, per RFC3339.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-time
+   * @default - the time stamp of the PutEvents call
+   */
+  readonly time?: string;
+}
+
+/**
+ * A EventBridge Pipes target that sends messages to an EventBridge event bus.
+ */
+export class EventBridgeTarget implements ITarget {
+  private eventBus: IEventBus;
+  private eventBridgeParameters?: EventBridgeTargetParameters;
+  public readonly targetArn: string;
+  constructor(eventBus: IEventBus, parameters?: EventBridgeTargetParameters) {
+    this.eventBus = eventBus;
+    this.targetArn = eventBus.eventBusArn;
+    if (parameters) {
+      this.eventBridgeParameters = parameters;
+      for (const validate of [validateDetailType, validateEndpointId, validateSource, validateTime]) {
+        validate(parameters);
+      }
+    }
+  }
+
+  grantPush(grantee: IRole): void {
+    this.eventBus.grantPutEventsTo(grantee);
+  }
+
+  bind(pipe: IPipe): TargetConfig {
+    if (!this.eventBridgeParameters) {
+      return {
+        targetParameters: {},
+      };
+    }
+
+    return {
+      targetParameters: {
+        inputTemplate: this.eventBridgeParameters.inputTransformation?.bind(pipe).inputTemplate,
+        eventBridgeEventBusParameters: this.eventBridgeParameters,
+      },
+    };
+  }
+}
+
+function validateDetailType({ detailType }: EventBridgeTargetParameters) {
```

Comment:
shouldn't all these `validateXxx` methods be worried about tokens? (and unit tests to check for that scenario)

Reply:
added